qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -217,30 +217,121 @@ def convert_acquire(self, shift, instruction):
                 })
         return self._qobj_model(**command_dict)
 
-    def convert_single_acquires(self, shift, instruction,
-                                qubits=None, memory_slot=None, register_slot=None):
-        """Return converted `AcquireInstruction`, with options to override the qubits,
-        memory_slot, and register_slot fields. This is useful for grouping
-        AcquisitionInstructions which are operated on a single AcquireChannel and
-        a single MemorySlot.
+    def convert_bundled_acquires(
+            self,
+            shift,
+            instructions_,
+    ):
+        """Bundle a list of acquires instructions at the same time into a single
+        Qobj acquire instruction.
 
         Args:
             shift (int): Offset time.
-            instruction (AcquireInstruction): acquire instruction.
-            qubits (list(int)): A list of qubit indices to acquire.
-            memory_slot (list(int)): A list of memory slot indices to store results.
-            register_slot (list(int)): A list of register slot addresses to store results.
+            instructions_ (List[Acquire]): List of acquire instructions to bundle.
         Returns:
             dict: Dictionary of required parameters.
+        Raises:
+            PulseError: If ``instructions`` is empty.
         """
-        res = self.convert_acquire(shift, instruction)
-        if qubits:
-            res.qubits = qubits
-        if memory_slot:
-            res.memory_slot = memory_slot
-        if register_slot:
-            res.register_slot = register_slot
-        return res
+        meas_level = self._run_config.get('meas_level', 2)
+
+        t0 = None
+        duration = None
+        memory_slots = []
+        register_slots = []
+        qubits = []
+        discriminators = []
+        kernels = []
+
+        if not instructions_:
+            raise PulseError('"instructions" may not be empty.')
+
+        for instruction in instructions_:
+            qubits.append(instruction.channel.index)
+
+            instr_t0 = shift + instruction.start_time
```

Comment:
This should move to line 238: `t0 = shift + instructions[0].start_time`, and then simply keep lines 255-257: `if instruction.start_time + shift != t0:`

Reply:
or you could loop over instructions right after 238 to check that `.start_time` is the same for all.

Reply:
Fixed in 5bbab0505.